source/mapping-rules/introduction.txt

```diff
@@ -63,29 +60,35 @@ You can define mapping rules for the following configurations:
 - Multiple tables that map into a single collection.
 
 Default Rules
--------------
+~~~~~~~~~~~~~
 
 Relational Migrator automatically creates a :ref:`New Documents mapping rule <rm-new-documents>`
 for each source table. This initial rule produces a MongoDB schema that
 is identical to the relational database. You can edit the initial rule 
 before running any data migration jobs.
 
 Data Type Detection
--------------------
+~~~~~~~~~~~~~~~~~~~
 
 Relational Migrator automatically selects the best MongoDB data type for each 
 source column.
 
 For example, SQL Server source data of types ``text``, ``varchar``,
 ``nvarchar`` or ``char``  will become the MongoDB ``string`` data type. 
 
-Preview Mapping Rules
----------------------
+Mapping Direction Preference
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. include:: /includes/mapping-options-preview.rst
+You can create a mapping rule based on a source :ref:`relational model <create-rule-relational>`
+or based on a destination :ref:`MongoDB model <create-rule-mongodb>`.
 
-Migrate as Options
-------------------
+The source relational model is static and unchanging while the target MongoDB model
+is dynamic and customizable. If your relational database schema changes, Relational Migrator 
+can refresh the relational model. For more information, 
+see :ref:`<rm-manage-relational-model>`.
+
+``Migrate as`` Options
+~~~~~~~~~~~~~~~~~~~~~~
 
 Mapping rules can be defined with the following ``Migrate as`` options:
```

Comment:
To avoid passive voice, maybe say "You can define mapping rules with the following..."